plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -2075,8 +2047,17 @@ protected StartAnswer execute(StartCommand cmd) {
             //
             // Setup ROOT/DATA disk devices
             //
+            if (Arrays.stream(sortedDisks).filter(disk -> disk.getType() == Volume.Type.ISO).count() > 1 && deployAsIs) {
```

Comment:
What about creating a method for this condition and improve readability?

Something like:
```
if (deployAsIs && multipleIsosAtached(sortedDisks))  { .. }
```